### DISCUSSION TITLE
text-by in version 1.6.1 does not work

### DISCUSSION THREAD

**Discussion by mariuscm** [epicmaxco/vuestic-ui#3117] [General]
Hello,

I've noticed that text-by in the va-select component stops working with vuestic-ui version 1.4.10

```
<template>
  <div style="max-width: 300px">
    <va-select
      v-model="value"
      :options="options"
      text-by="value"
      value-by="id"
    />
  </div>
</template>
<script>
export default {
  data() {
    return {
      value: "",
      options: [
        { id: 1, value: "one" },
        { id: 2, value: "two" },
        { id: 3, value: "three" },
      ],
    };
  },
};
</script>
```

With the code above, when I select one of the options, I expect the va-select to display the value of my selection (one, two, or three). What happens is that the va-select displays the corresponding id (1, 2, 3).

If I downgrade to 1.4.9 and below this works as expected, but then I want to use components from 1.5.0, specifically the va-tree-view.

Please let me know what I can do to fix this. 

Thanks,
Marius

**Comment by m0ksem** [comment]
This must be fixed after this PR in next release.
https://github.com/epicmaxco/vuestic-ui/pull/3092

I just need to find power to review this... But this is what we want to fix in 1.6.2 100%.
